extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -125,60 +122,78 @@ export class AmpAdXOriginIframeHandler {
       this.sendEmbedInfo_(this.baseInstance_.isInViewport());
     }));
 
+    // Iframe.onload normally called by the Ad after full load.
+    const iframeLoadPromise = loadPromise(this.iframe).then(() => {
```

Comment:
Possible. But in reality this almost never happens. The only real situation I saw is when `render-start` and `onload` are triggered at the same time. In that case, `onload` arrives a few milliseconds earlier.